core/src/main/java/com/orange/ouds/core/utilities/Preconditions.kt

```diff
@@ -0,0 +1,68 @@
+/*
+ * Software Name: OUDS Android
+ * SPDX-FileCopyrightText: Copyright (c) Orange SA
+ * SPDX-License-Identifier: MIT
+ *
+ * This software is distributed under the MIT license,
+ * the text of which is available at https://opensource.org/license/MIT/
+ * or see the "LICENSE" file for more details.
+ *
+ * Software description: Android library of reusable graphical components
+ */
+
+package com.orange.ouds.core.utilities
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.alpha
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalInspectionMode
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import com.orange.ouds.core.theme.LocalColoredBox
+import com.orange.ouds.core.theme.OudsTheme
+import com.orange.ouds.core.theme.dashedBorder
+import com.orange.ouds.foundation.LocalSnapshotTest
+
+@Composable
+internal fun CheckState(
```

Comment:
As discussed together we have a new name: `CheckedContent` \o/
And maybe we can rename the file.